### ISSUE TITLE
Colors in summary are bugged when assert_almost_eq fails

### ISSUE THREAD

**Issue by Ajver** [bitwes/Gut#149]
When I created `assert_almost_eq` test and it failed, the yellow color from the end of the line is not overridden and all next text is yellow as well (as shown on the screenshot).

![image](https://user-images.githubusercontent.com/40212297/77767282-1eca9700-7041-11ea-985d-a74e4d29b503.png)

Steps to reproduce:

1. Create any test and make an assertion: `assert_almost_eq`, that fails.
1. Enjoy

**Comment by bitwes** [comment]
Good catch.  I've seen this happen every now and again but hadn't tracked it down.  I leverage syntax. highlighting in the control and not any explicit start/stop for colors.  Some value is probably being escaped or some default behavior of the control is poking through.  I don't know if the control supports different formatting now.

**Comment by bitwes** [comment]
In looking into this and thinking about how to improve the scene's output I've rediscovered some of my initial issues.

TLDR a block comment format start or end string is breaking in the TextEdit syntax highlighting.

## TextEdit vs RichTextLabel
TextEdit uses a syntax highlighting approach for colors and the RichTextLabel uses tags and a push/pop mechanism to color output.  

RichTextLabel auto-wraps text and it doesn't appear you can disable it.  I don't really like the output wrapped.

TextEdit allows for coloring output AFTER it has been displayed.  This is done with the test names.  I print the name first so if anything breaks you can see the current test name.  Once the test passes or fails I add the test name to the proper list and it gets colored green or red.

I guess this should just be fixed instead of reworked like I wanted to.
